test.html

```diff
@@ -17,6 +17,7 @@
   <link href="styles/expandingSection.css" rel="stylesheet"/>
 </head>
 <body>
+  <p></p>
   <header class="header grid-header-container">
```

Comment:
This empty paragraph tag should not get merged into upstream. Can you remove it please?